ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstringTest.cs

```diff
@@ -0,0 +1,17 @@
+using LeetCodeTraining.LongestSubstring1;
+
+namespace LeetCodeTraining.Tests
+{
+    public class LongestSubstringSolutionTest
+    {
+        [Theory]
+        [InlineData("abcabcbb",3)]
+        [InlineData("bbbbb", 1)]
+        [InlineData("pwwkew", 3)]
+        public void LongestSubstring(string s,int expectedResults)
+        {
+            int Result = SubstringSolution.LengthOfLongestSubstring(s);

```

Comment:
Please add //Act and //Assert comments. and add line between blocks to read the code easier.